package.json

```diff
@@ -5,17 +5,17 @@
   "homepage": "https://kg.humanbrainproject.org/webapp",
   "proxy": {
     "/proxy": {
-      "target": "https://kg.humanbrainproject.org",
```

Comment:
Is there a way to remove this hbp-specific configuration from the public code repository?

Reply:
Not easily.
We could add another package.json in the main gitlab repository and use that one to run the code locally. But then we would need to manually sync the 2 packages.json to use the same dependencies.
Another solution would be to to run a proxy server on the local machine